app/isomorphic/components/amazon-polly/amazon-polly.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import get from "lodash/get";
+import { HeadsetIcon } from "../../../assets/icons/headset";
+import { CloseIcon } from "../../../assets/icons/close";
+
+import "./amazon-polly.m.css";
+
+export class AmazonPolly extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false
+    };
+  }
+
+  onClickHandler = () => {
+    this.setState({
+      isOpen: true
+    })
+  }
+
+  closeHandler = () => {
+    this.setState({
+      isOpen: false
+    })
+  }
+
+  render() {
+    const { caption, story, config, audioAttributes = { controls: true }, headsetColor = "#333", headsetSize = 24 } = this.props;
```

Comment:
i would suggest passing only the thing which is not a context. like story, audioAttributes, caption. It would be better if we can read the config from the store. The lesser the caller needs to know about inner workings of the component, the better it is.